src/bicep/modules/firewall.bicep

```diff
@@ -132,6 +134,35 @@ resource firewallNetworkRuleCollectionGroup 'Microsoft.Network/firewallPolicies/
         name: 'AllowAzureCloud'
         priority: 100
       }
+      {
+        ruleCollectionType: 'FirewallPolicyFilterRuleCollection'
+        action: {
+          type: 'Allow'
+        }
+        rules: [
+          {
+            ruleType: 'NetworkRule'
+            name: 'AllSpokeTraffic'
+            ipProtocols: [
+              'Any'
+            ]
+            sourceAddresses: [
+              firewallSupernetIPAddress
+            ]
+            sourceIpGroups: []
+            destinationAddresses: [
+              '*'
```

Comment:
Shoud this be the suprnetIpAddress range versus '*'? I believe right now this would technically allow all outbound traffic from the spokes, no?

Reply:
My thoughts exactly - this change allows all traffic within the VNets to access any address, including external, which breaks the "all other traffic is restricted by default" statement in https://github.com/Azure/missionlz/blob/main/README.md
@glennmusa @haithamshahin333 this looks like an unintended consequence of this change.